### PR TITLE
feat(leadspace-block): leadspace-block web component

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

Web Component: Develop Lead Space Block of the React version - Group 3 #3753

### Description

Leadspace Block web component

screencapture-localhost-9000-2020-11-23-17_44_26

### Changelog

**New**

- leadspace-block web component

**Changed**

- styles and selectors for web component in leadspace-block style sheet

### CODE REVIEW

File: packages/web-components/src/components/leadspace-block/__stories__/README.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+See [our README](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components#stable-selectors-for-analytics-and-integratione2e-testing-in-web-components) to see how Web Components selector and `data-autoid` should be used.
+
+| Web Components selector   | Compatibility selector                 | Description |
+| ------------------------- | -------------------------------------- | ----------- |
+| `<dds-leaving-ibm-modal>` | `data-autoid="dds--leaving-ibm-modal"` | Component   |
+| `dds--leaving-ibm-cta`    | `data-autoid="dds--leaving-ibm-cta"`   | Interactive |
+
+## Props
+
+<Props of="dds-leaving-ibm-composite" />
```
Comment (asudoh) [inline packages/web-components/src/components/leadspace-block/__stories__/README.stories.mdx:39]: Meant to update here...?

**Review by asudoh — COMMENTED** [review]
Nice progress @annawen1!

**Review by IgnacioBecerra — APPROVED** [review]
LGTM, thanks @annawen1! Very tiny nitpick, the React version has a different icon for "View all products".
Screen Shot 2020-11-24 at 1 16 54 PM

Doesn't matter too much since it's customizable after all, but it may good to match the React Storybook as closely as possible. Thanks!

**Comment by annawen1** [comment]
> LGTM, thanks @annawen1! Very tiny nitpick, the React version has a different icon for "View all products".
> Screen Shot 2020-11-24 at 1 16 54 PM
> 
> Doesn't matter too much since it's customizable after all, but it may good to match the React Storybook as closely as possible. Thanks!

Gotcha, I've changed the last icon to the download one!

**Pull request merged**